Automattic/wp-calypso
# Style selection: Site preview

#### Proposed Changes

This PR implements the site preview component in the full-screen theme preview. See screenshot below for reference:
![Screen Shot 2022-09-06 at 10 48 40 AM](https://user-images.githubusercontent.com/797888/188536661-422d02c2-8acc-4e37-9fc8-f95f0a66f78d.png)

In addition, this PR also implements the style switching of the site preview, controlled via style switcher buttons in the full-screen preview's sidebar. See recording below for reference:
https://user-images.githubusercontent.com/797888/188536872-99883154-a90c-44fa-8ccf-518d5bb25159.mp4

Known issues as tracked in #67388 

#### Testing Instructions



* Head to the unified design picker `/setup/designSetup?siteSlug=${site_slug}`
* Click on any static theme and ensure that the new full-screen theme preview shows up.
* Ensure that the site preview device switcher works as intended.
* Ensure that the site preview style switching works as intended.
* Click on any generated design and ensure that the old full-screen theme preview shows up.

#### Pre-merge Checklist






Related to #66741
